packages/react-theming/README.md

```diff
@@ -1,6 +1,11 @@
 # Summary
 
-`react-compose` provides a set of tools for creating themable components. The `compose` tool takes a functional component and, given default options and contextual overrides provided through React context, computes injected props including `classes`, `slots`, and `slotProps`. Styles are computed only when encountering unique theme objects per component, resulting in optimized performance.
+`react-compose` provides a set of tools for creating themable
+components. The `compose` tool takes a functional component and, given
+default options and contextual overrides provided through React
+context, computes injected props including `classes`, `slots`, and
+`slotProps`. Styles are computed only when encountering unique theme
```

Comment:
Aren't we going to re-calculate for different props combination? How can only the theme be key for the cache? Based on what was priviosly written:

>  A variant is defined as an additional prop on the created component which has
the ability to adjust the look and feel of the component.

it seems that we will need to include the props combination in the cache.

Reply:
Yes, this is a good point. This work needs to be done still however, these docs are just documenting the existing code.

Reply:
What existing code is that? I thought we are defining the vision of the principles and createComponent. Is this a doc for the current compose/createComponent that is part of the react-theming mechanism?

Reply:
Yes, sorry for muddying the waters. I will remove the README changes from this PR so it's more focused.

Reply:
I see now, I thought everything is part of the component principles write up. Yep, it would be best if we can extract it. Thanks!